lib/call.js

```diff
@@ -77,21 +80,33 @@ var Call = function (client) {
 	 * formated number,like +19195551212, or a valid SIP URI, like sip:someone@somewhere.com).
 	 * @param {String} [params.to] The number to filter calls that was called to (must be either an E.164
 	 * formated number,like +19195551212, or a valid SIP URI, like sip:someone@somewhere.com).
-	 * @param {Number} [params.page=0] Used for pagination to indicate the page requested for querying a list of calls.
-	 * If no value is specified the default is 0.
-	 * @param {Number} [params.size=25] Used for pagination to indicate the size of each page requested
-	 * for querying a list of calls. If no value is specified the default value is 25 (maximum value 1000).
 	 * @param {Function} callback A callback with the list of calls
 	 * @return {Promise} A promise for the list of calls
 	 */
 	this.list = function (params, callback) {
+		var self = this;
 		return client.makeRequest({
 			path   : "calls",
 			method : "GET",
 			qs     : params
 		})
 		.then(function (response) {
-			return response.body;
+			var callListResponse = {
+				calls       : response.body,
+				hasNextPage : false,
+				getNextPage : function (nextCallback) {
+					return Promise.reject("Next page does not exist.")
+					.asCallback(nextCallback);
+				}
+			};
+			var nextLink = getNextLink(response.headers);
+			if (nextLink) {
+				callListResponse.hasNextPage = true;
+				callListResponse.getNextPage = function (nextCallback) {
+					return self.list(nextLink, nextCallback);
+				};
+			}
+			return callListResponse;
```

Comment:
The reason why it is structured that way is because we want to have some default behavior (https://github.com/bandwidthcom/node-bandwidth/pull/54#discussion_r69158066).